### DISCUSSION TITLE
Elementary OS 6 Odin don’t boot with USB - Error: Failed To Open EFI/BOOT

### DISCUSSION THREAD

**Discussion by fejabame** [elementary/os#543] [Q&A] [answered]
Hi, I'm new to the Forum

I'm trying to install Elementary OS 6 Odin on my PC but I get error

Do you know how to fix it? Thank you

ERROR:
Booting in insecure mode
Failed to open EFi/BOOT/ Invalid Parameter
Failed to load image EFI/BOOT/ Invalid Parameter
Star_image returned invalid parameter

**Comment by cassidyjames** [comment, marked as answer]
Thanks for posting! It looks like you may be hitting [this issue](https://github.com/elementary/triage/issues/74) which is noted in the [FAQ](https://github.com/elementary/os/wiki/elementary-OS-6-Odin-FAQ) listed on the homepage, installation instructions, and announcement post.

We're currently in the process of releasing a new ISO file which should resolve this issue. You can either try the workarounds listed on the above issue, or wait about a day to download the new version once it's listed on our site; we'll also send out an update on our [Twitter profile](https://twitter.com/elementary) and the [release tag](https://github.com/elementary/os/releases/tag/6.0-20210823) once the new ISO is available.
